demo/cluster/aws-standalone-istio-child.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: k0rdent.mirantis.com/v1alpha1
+kind: ClusterDeployment
+metadata:
+  name: aws-ue2-child-istio
+  namespace: kcm-system
+spec:
+  template: aws-standalone-cp-0-1-2
+  credential: aws-cluster-identity-cred
+  config:
+    clusterIdentity:
+      name: aws-cluster-identity
+      namespace: kcm-system
+    controlPlane:
+      instanceType: t3.large
+    controlPlaneNumber: 1
+    publicIP: false
+    region: us-east-2
+    worker:
+      instanceType: t3.medium
+    workersNumber: 3
+    clusterLabels:
+      k0rdent.mirantis.com/istio-role: child
```

It still requires labels as in [aws-standalone-child.yaml](https://github.com/k0rdent/kof/blob/1302e960958e00e79ddf1f74ab5bea0b48e6ae46/demo/cluster/aws-standalone-child.yaml#L21-L24), right?
```suggestion
      k0rdent.mirantis.com/istio-role: child
      k0rdent.mirantis.com/kof-storage-secrets: "true"
      k0rdent.mirantis.com/kof-cluster-role: child
      k0rdent.mirantis.com/kof-regional-domain: aws-ue2.kof.example.com
```